### ISSUE TITLE
ImportError: No module named 'IPython'

### ISSUE THREAD

**Issue by madnight** [theskumar/python-dotenv#64]
Since version `python-dotenv-0.7.0` (version `python-dotenv-0.6.5` worked)

```
File "/usr/lib/python3.5/site-packages/dotenv/__init__.py", line 3, in <module>
    from .ipython import load_ipython_extension
  File "/usr/lib/python3.5/site-packages/dotenv/ipython.py", line 4, in <module>
    from IPython.core.magic import Magics, magics_class, line_magic
ImportError: No module named 'IPython'
```

**Comment by theskumar** [comment]
@madnight thanks for reporting this i'm looking into it.

**Comment by hellysmile** [comment]
please, do not make ipython strict requirement.

Also, can You make click support to be optional?

**Comment by theskumar** [comment]
@madnight Fixed in 0.7.1. Can you please check and confirm. 

@hellysmile iPython is made optional. can you create an issue for click, will see what what we can do about it.
